content/chainguard/chainguard-images/working-with-images/image-stigs.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "STIGs for Chainguard Images"
+linktitle: "STIGs"
+type: "article"
+description: "A conceptual overview of STIGs, which are available for Chainguard Images."
+date: 2024-06-13T15:56:52-07:00
+lastmod: 2024-06-13T15:56:52-07:00
+draft: false
+tags: ["IMAGES", "PRODUCT", "CONCEPTUAL"]
+images: []
+menu:
+  docs:
+    parent: "concepts"
+weight: 055
+toc: true
+---
+
+
+The practice of using STIGs to secure various technologies originated with the United States Department of Defense (DoD). If an organization uses a certain kind of software, say MySQL 8.0, they must ensure that their implementation of it meets the requirements of the [associated STIG](https://public.cyber.mil/announcement/stig-update-disa-has-released-the-oracle-mysql-8-0-stig/) in order to qualify as a vendor for the DoD. More recently, other compliance frameworks have begun acknowledging the value of STIGS, with some going so far as to require the use of STIGs in their guidelines.
+
+[Chainguard announced](https://www.chainguard.dev/unchained/stig-hardening-container-images) the release of a General Purpose Operating System STIG with the goal that it will help customers integrate Chainguard Images into their workflows with the confidence that they're doing so securely. This conceptual article aims to give a brief overview of what STIGs are and how they can be valuable in the context of container images.
```

Comment:
```suggestion
[Chainguard announced](https://www.chainguard.dev/unchained/stig-hardening-container-images) the release of a STIG for the General Purpose Operating System (GPOS) SRG with the goal that it will help customers integrate Chainguard Images into their workflows with the confidence that they're doing so securely. This conceptual article aims to give a brief overview of what STIGs are and how they can be valuable in the context of container images.
```